extension/chrome/src/js/injector_loader.js

```diff
@@ -3,11 +3,11 @@ const extensionId = chrome.runtime.id || 'stashFF';
 
 function injectEngine(){
 	const manifest = chrome.runtime.getManifest();
-	createInlineScript("var stashRGEVersion = '" + manifest.version + "'; var chromeExtId='" + extensionId + "'; stashIcon='"+chrome.extension.getURL('img/stash128.png')+"';");
+	createInlineScript(`var stashRGEVersion = '${manifest.version}'; var chromeExtId='${extensionId}'; stashIcon='${chrome.extension.getURL('img/stash128.png')}';`);
```

Comment:
how is this string eval better from just running `window.stashRGEVersion  = ...` plz? is webextension's `window` separated from the page?

Reply:
Ummm ... probably. I did not change this logic other than eslint autofix. Maybe @dragouf knows?

Reply:
oh, just eslint autofix. ok.